frontend/public/components/dashboards-page/overview-dashboard/overview-dashboard.tsx

```diff
@@ -0,0 +1,65 @@
+import * as React from 'react';
+import Measure from 'react-measure';
+import { Grid, GridItem } from '@patternfly/react-core';
+
+import { Dashboard } from '../../dashboard';
+import { HealthCard } from './health-card';
+
+export const MEDIA_QUERY_LG = 990;
+
+export class OverviewDashboard extends React.Component<{}> {
+  state = {
+    dimensions: {
+      width: -1,
+    },
+  };
+
+  onResize = contentRect => {
+    this.setState({ dimensions: contentRect.bounds });
+  };
+
+  render() {
+    const { width } = this.state.dimensions;
+    let grid;
+    if (width <= MEDIA_QUERY_LG) {
```

Comment:
CSS media queries wont tell us the dimensions of the component which differ when side menu (nav) is closed or open. On some screens it can make quite a difference. We also have a Utilization card which is changing layout based on card size https://docs.google.com/document/d/1q0DIcUKgFhvNbtGt-SMl35Z2jHA_QO2r7aaBuFmwE2U/edit#heading=h.dforzl394dbm and we use react-measure there too.